app-misc/hyfetch/hyfetch-1.4.11-r1.ebuild

```diff
@@ -20,10 +21,27 @@ fi
 LICENSE="MIT"
 SLOT="0"
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.4.11-pyproject.patch
+	"${FILESDIR}"/${PN}-1.4.11-neofetch.patch
+)
+
 RDEPEND="
 		dev-python/typing-extensions[${PYTHON_USEDEP}]
 "
 
+python_install() {
+	newbashcomp hyfetch/scripts/autocomplete.bash ${PN}
+	newzshcomp hyfetch/scripts/autocomplete.zsh _${PN}
+
+	distutils-r1_python_install
+
+	dodir /usr/bin/
+	mv neofetch "${D}/usr/bin/neowofetch" || die
```

Comment:
```
>>> Install app-misc/hyfetch-1.4.11-r1 into /var/tmp/portage/app-misc/hyfetch-1.4.11-r1/image
 * python3_10: running distutils-r1_run_phase python_install
 * python3_11: running distutils-r1_run_phase python_install
mv: cannot stat 'neofetch': No such file or directory
 * ERROR: app-misc/hyfetch-1.4.11-r1::gentoo failed (install phase):
 *   (no error message)
```
I **believe** this is a problem with multiple python_compat enabled (like 11 and 12) - you move the binary out of work dir when installing files for python-3.11, so it doesn't exist for python-3.12. I'll change it to **cp** instead.